content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1790,6 +1790,22 @@ uses this annotation.
 See [annotations](https://kubernetes-sigs.github.io/aws-load-balancer-controller/latest/guide/service/annotations/)
 in the AWS load balancer controller documentation.
 
+### service.beta.kubernetes.io/load-balancer-security-group {#service-beta-kubernetes-io-aws-load-balancer-security-groups}
+
+Example: `service.beta.kubernetes.io/aws-load-balancer-security-groups: "sg-53fae93f, sg-8725gr62r"`
+
+Used On: Service
+
+The cloud controller manager integration with AWS elastic load balancing configures
+this annotation to specify a comma seperated list of frontend securityGroups you want
+to attach to an NLB. Both name and ID of securityGroups are supported where name matches
+a `Name` tag, not the `groupName` attribute.
+
+When not present, the controller will automatically create one security group.
+When present, all other previously assigned security groups are replaced by this list.
+Also you need to configure the security groups on your target instances/ENIs to allow
+inbound traffic from the load balancer.
+
```

Comment:
The annotation has been removed in v1.27.
- https://github.com/kubernetes/kubernetes/pull/115838

We'd better add a reminder here.
```suggestion

{{< note >}}
Starting in v1.27, this annotation was removed.
{{< /note >}}

```
/cc @torredil  @dims  for confirm

Reply:
should i remove this information here:
```
Also you need to configure the security groups on your target instances/ENIs to allow
inbound traffic from the load balancer.
``` 
And mention the note? I'll rather add a link to the documentation. WDYT?

Reply:
This note is not necessary. We already have a note before this one.

Reply:
Noted.

Reply:
See https://github.com/kubernetes/website/pull/43009#discussion_r1332168164